tools/tidb-binlog-cluster.md

```diff
@@ -0,0 +1,620 @@
+---
+title: TiDB-Binlog Cluster User Guide
+summary: This document introduces TiDB-Binlog of the cluster version, including its architecture, deployment, operations and monitoring.
```

Comment:
Please refer to other documents for the summary sentence pattern. Only several specific sentence patterns are used.